REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.4
-Compat 0.7.15
+Compat 0.9.4
```

Comment:
nevermind, you are calling `xor` in a test so this is right, but it was introduced in the previous version